examples/setrise/main.c

```diff
@@ -5,7 +5,7 @@
 #include <predict/predict.h>
 #include <predict/observer.h>
 
-double observer_next_sunset(const observer_t *observer, double time)
+double observer_next_sunset(const observer_t *observer, double time, struct observation *obs)
```

Comment:
Yes, plan was to move this function to the library when working correctly. I suggest we create a new issue for this feature and continue development in a new branch.